### PR TITLE
Added <summary> to GetCARoots, added GetCARoots with QueryOptions parameter

### PR DESCRIPTION
**Pull request by IvanKolchanov**

### CODE REVIEW

**Comment by IvanKolchanov** [comment]
@marcin-krystianc the PR with fixes to #332

File: Consul/Agent.cs [hunk 1]
```diff
@@ -1164,11 +1164,27 @@ public async Task<QueryResult<ServiceConfiguration>> GetServiceConfiguration(str
...
         }
 
+        /// <summary>
+        /// GetCARoots returns root certificates in the cluster
+        /// </summary>
+        /// <param name="ct">Cancellation Token</param>
+        /// <returns>Root certificates</returns>
         public async Task<QueryResult<CARoots>> GetCARoots(CancellationToken ct = default)
         {
             return await _client.Get<CARoots>("v1/agent/connect/ca/roots", QueryOptions.Default).Execute(ct).ConfigureAwait(false);
```
Comment (marcin-krystianc) [inline Consul/Agent.cs:1174]: Can you call the 
```suggestion
            return await GetCARoots(QueryOptions.Default, ct);
```

Comment (IvanKolchanov) [reply Consul/Agent.cs:1174]: Oh, makes sense. Shouldn't it be "return await GetCARoots(QueryOptions.Default, ct).ConfigureAwait(false);" though?

Comment (marcin-krystianc) [reply Consul/Agent.cs:1174]: yeah, you are right.

**Review by marcin-krystianc — APPROVED** [review]
Thanks!

**Pull request merged**